openpnm/algorithms/GenericTransport.py

```diff
@@ -311,8 +313,22 @@ def set_rate_BC(self, pores, values, mode='merge'):
         The definition of ``quantity`` is specified in the algorithm's
         ``settings``, e.g. ``alg.settings['quantity'] = 'pore.pressure'``.
         """
+        # support 'values' keyword
+        if 'values' in kwargs.keys():
+            rates = kwargs.pop("values")
+            warnings.warn("'values' has been deprecated, use 'rates' instead.",
+                          DeprecationWarning)
+        # handle total_rate feature
+        if total_rate is not None:
+            if np.isscalar(total_rate) is not True:
```

Comment:
I think it should be `if not np.isscalar(total_rate):`. It's so-called more "Pythonic" since `isscalar` already has the verb "is", and maybe more importantly since `if bool:` already evaluates to True/False in Python, so no need to explicitly compare, e.g. `if bool is True:`, for instance.

I know it's probably a bit annoying, but that's the recommended approach (you can read up [here](https://docs.quantifiedcode.com/python-anti-patterns/readability/comparison_to_true.html) for a complete style guide if you'd like)

Reply:
Hey @ma-sadeghi no problem. I changed it. If Jeff is happy, all that's left is to clear the codecov error, which I will need you to do.

Reply:
code cov is fine, it's the codefactor thing that is a pain.